### DISCUSSION TITLE
Command Get-AdminPowerAppCdsAdditionalNotificationEmails cannot be executable

### DISCUSSION THREAD

**Discussion by mkuleta** [PowerShell/PowerShell#21390] [General]
![image](https://github.com/PowerShell/PowerShell/assets/56870032/4751b746-6063-4dd5-8a4c-01e83e8d2843)
I used find funcktion after installig package by command 
Install-Module -Name Microsoft.PowerApps.Administration.PowerShell -RequiredVersion 2.0.180

and now when i want to use Get-AdminPowerAppCdsAdditionalNotificationEmails
i recieved below error 
![image](https://github.com/PowerShell/PowerShell/assets/56870032/50a98b67-c613-4c04-a31c-d5634e1e1a08)

**Comment by rhubarb-geek-nz** [comment]
The page at [PSGallery](https://www.powershellgallery.com/packages/Microsoft.PowerApps.Administration.PowerShell/2.0.180) refers to [Project Site](https://learn.microsoft.com/en-us/power-platform/admin/powerapps-powershell)

That says

> The modules described in this document use .NET Framework, which is incompatible with PowerShell 6.0 and later. These later versions use .NET Core.

**Comment by mkuleta** [comment]
But im using PowerShell in version showed below:
![image](https://github.com/PowerShell/PowerShell/assets/56870032/66a64127-007c-467d-84f1-b8df8bb474eb)

**Comment by rhubarb-geek-nz** [comment]
Thank you for clearing that up. These projects here are for the open source PowerShell Core, versions 6.0 upwards, hence why I was checking.
